examples/vue/src/main.ts

```diff
@@ -1,7 +1,6 @@
-import { createApp } from "vue";
-import App from "./App.vue";
-import router from "./router";
+import { createApp } from 'vue';
+import App from './App.vue';
+import router from './router';
+import './styles.scss';
```

Comment:
There are styles in the next example, that I copied over to the Vue example so the look and feel would match.

Reply:
Gotcha, I'll probably have to do this in Angular too... and hope Angular is cooperative with scss.

Reply:
Do you recall which primitives needed this scss file? I suppose this is filling in the gap of not having primitives on Angular/Vue.

Reply:
All the ones with the !import were used for sure. Also I added a font.